docs/artifacts.md

```diff
@@ -134,3 +135,31 @@ You can use getByName() method to search database for artifacts using the name a
 import gem5art.artifact
 for i in gem5art.artifact.getByName("gem5"):print(i)
 ```
+
+## Artifacts API Documentation
+```eval_rst
+Artifact
```

Comment:
Can you make this more descriptive? Maybe Artifact Module?

Reply:
Ok!